--- origsrc/pitivi-0.13.3/pitivi/application.py	2009-09-12 04:10:15.000000000 -0500
+++ src/pitivi-0.13.3/pitivi/application.py	2009-11-03 22:00:46.727497100 -0600
@@ -38,7 +38,6 @@ from gettext import gettext as _
 import pitivi.instance as instance
 
 from pitivi.check import initial_checks
-from pitivi.device import get_probe
 from pitivi.effects import Magician
 from pitivi.configure import APPNAME
 from pitivi.settings import GlobalSettings
@@ -133,7 +132,6 @@ class Pitivi(Loggable, Signallable):
             self.settings.get_local_plugin_path(),
             self.settings.get_plugin_settings_path())
         self.effects = Magician()
-        self.deviceprobe = get_probe()
 
         self.projectManager = ProjectManager()
         self._connectToProjectManager(self.projectManager)
@@ -161,8 +159,6 @@ class Pitivi(Loggable, Signallable):
             return False
         self.threads.stopAllThreads()
         self.settings.storeSettings()
-        self.deviceprobe.release()
-        self.deviceprobe = None
         self.current = None
         instance.PiTiVi = None
         self.emit("shutdown")
--- origsrc/pitivi-0.13.3/pitivi/ui/mainwindow.py	2009-09-12 04:10:15.000000000 -0500
+++ src/pitivi-0.13.3/pitivi/ui/mainwindow.py	2009-11-03 22:01:11.890341000 -0600
@@ -202,10 +202,7 @@ class PitiviMainWindow(gtk.Window, Logga
         self.app.action_log.connect("cleaned", self._actionLogCleaned)
 
         # if no webcams available, hide the webcam action
-        self.app.deviceprobe.connect("device-added", self._deviceChangeCb)
-        self.app.deviceprobe.connect("device-removed", self._deviceChangeCb)
-        if len(self.app.deviceprobe.getVideoSourceDevices()) < 1:
-            self.webcam_button.set_sensitive(False)
+        self.webcam_button.set_sensitive(False)
 
         self.show()
 
